Gemfile

```diff
@@ -57,6 +57,8 @@ group :development, :test do
   gem 'pry'
   gem 'simplecov'
   gem 'shoulda-matchers'
+  gem 'factory_bot_rails'
```

Comment:
For IC's I would not spend time setting up `faker` or `factory bot`. We want to see that you can complete the US's and their functionality.